cmake/CmakeFunctionHelper.cmake

```diff
@@ -158,10 +158,10 @@ elseif(${TARGET} STREQUAL "AMD_GPU")  # need investigation
     endif()
   endif()
 else() # default cpu backend
-  set(gemm_configuration_0 "float"  64 "false" "false" "false" 64 8 8 8 8 1 1 "no_local" "naive")
-  set(gemm_configuration_1 "float"  64 "false" "false" "false" 64 8 8 8 8 1 1 "no_local" "standard")
-  set(gemm_configuration_2 "double" 64 "false" "false" "false" 64 8 8 8 8 1 1 "no_local" "naive")
-  set(gemm_configuration_3 "double" 64 "false" "false" "false" 64 8 8 8 8 1 1 "no_local" "standard")
+  set(gemm_configuration_0 "float"  64 "false" "false" "false" 64 8 8 8 8 1 1 "no_local" "naive" 8)
```

Comment:
I assume you have added the vectorize size here for the future when you vectorize the no local version?

Reply:
Yes as with the tall skinny all the gemm methods involved in calling the kernel need to support these extra parameters even if the kernel doesn't use them yet.